jenkins.check_copyright.sh

```diff
@@ -32,7 +32,106 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testBitwise.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testCalls.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testComparison.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testComparisonSelect.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testConstructorLanes.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testConversion-2.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testConversion.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testLoadStore.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testMinMax.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testMisc.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testMul.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testShift.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testShuffle.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testSplat.js$' |
+    grep -v -E 'test/SIMD.uint16x8.asmjs/testSwizzle.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testAddSub.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testBitwise.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testCalls.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testComparison.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testComparisonSelect.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testConstructorLanes.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testConversion-2.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testConversion.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testLoadStore.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testMinMax.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testMisc.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testMul.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testShift.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testShuffle.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testSplat.js$' |
+    grep -v -E 'test/SIMD.uint32x4.asmjs/testSwizzle.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testAddSub.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testAddSubSaturate.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testBitwise.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testCalls.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testComparison.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testComparisonSelect.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testConstructorLanes.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testConversion-2.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testConversion.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testLoadStore.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testMinMax.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testMisc.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testMul.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testShift.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testShuffle.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testSplat.js$' |
+    grep -v -E 'test/SIMD.uint8x16.asmjs/testSwizzle.js$' |
```

Comment:
Perhaps a subdirectory? I agree there's not really a pretty solution besides altering the check. In any case, we're still waiting to hear what Microsoft legal recommends for these cases.